samples/grid-lite/demo/grid-theming/demo.html

```diff
@@ -0,0 +1,8 @@
+<script src="https://code.highcharts.com/grid/grid-lite.js"></script>
+
+<div id="container">
+    <div id="table_0" class="table"></div>
+    <div id="table_1" class="table"></div>
+    <div id="table_2" class="table"></div>
+    <div id="table_3" class="table"></div>
```

Comment:
Can we use a different name for a class?

The class name should not be the same as a HTML tag.

Reply:
`table-wrapper` used.